### DISCUSSION TITLE
J1010 Boot From SD Card | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#249] [Q&A]
# J1010 Boot From SD Card | Seeed Studio Wiki

J1010 Boot From SD Card

https://wiki.seeedstudio.com/J1010_Boot_From_SD_Card/

**Comment by PHCLenzi** [comment]
It became quite difficult to understand what needs to be done to switch back and forth from
  boot from SD card to eMMC. I am working with the dev kit with the J1010 base board and don't have the path "/media/seeed/...".

1 - why figures 4 and 5 are the same? just to say that the two boot files, SD card and eMMC, are the same?

2 - Why does it show in figure 3 that after formatting, the contents of the SD card are mounted at /media/seeed/<name>? If I format the SD card in another ubuntu system will it make any difference?

**Comment by MatthewJeffson** [comment]
I think maybe you should [Flash Jetpack](https://wiki.seeedstudio.com/reComputer_J1010_J101_Flash_Jetpack/) to the J1010 first? I am not quite sure, though. You can email our professional tech support: fae@seeed.cc
